### ISSUE TITLE
kubecf.features.suse_buildpacks.enabled false does not set cflinuxfs3 as default - 2.2.0

### ISSUE THREAD

**Issue by kramerul** [cloudfoundry-incubator/kubecf#841]
**Describe the bug**

When installing kubecf 2.2.0 with `kubecf.features.suse_buildpacks.enabled` set to `false` `cf push` will no longer work.

It fails with

```
Default stack with name 'sle15' not found
...
CF-MissingDefaultStackError
```

**To Reproduce**

* Install kubecf with `kubecf.features.suse_buildpacks.enabled = false`
* Run `cf push` with dora app

**Expected behavior**

`cf push` works

**Environment**
 - CF Version
 - Component/Plugin Name
 - Others

**Additional context**

**Comment by fargozhu** [comment]
@thardeck @f0rmiga wouldn't be better to have a more explicit way to declare it like: 

`kubecf->features->suse->buildpacks->enabled`
or
`kubecf->features->buildpacks->suse->enabled`

**Comment by f0rmiga** [comment]
@kramerul I assume you are using Eirini, right? We only make the `sle15` stack as default with Eirini. Ping @viovanov.

**Comment by jandubois** [comment]
> We only make the `sle15` stack default with Eirini.

This happened in d67c8e9. If we only support one stack with eirini, then we should also disable setting `kubecf.features.suse_buildpacks.enabled` to `false` for eirini as well, to enforce consistency. Or maybe better throw an error instead, because we are now ignoring an explicit user choice.

Of course best would be if disabling the SUSE buildpacks would switch the eirinifs to a cflinuxfs3 based rootfs.

**Comment by fargozhu** [comment]
> @kramerul I assume you are using Eirini, right? We only make the `sle15` stack default with Eirini. Ping @viovanov.

but we should switch to cflinuxfs3 as default in that case

**Comment by jandubois** [comment]
> but we should switch to cflinuxfs3 as default in that case

Yes, but I think this is slightly complicated, as the eirinifs is included in the bits-service image, so you would have to also configure a different image. I haven't tested it, but I would assume this would work (with the 2.2.0 release):

```
--set features.suse_buildpacks.enabled=false --set bits.images.bits_service=eirini/bits-service:2.36.0
```

You would still have to use `cf push -s cflinuxfs3` because we unconditionally set the stack name for eirini right now.

**Comment by kramerul** [comment]
@fargozhu , yes we are using Eirini.

**Comment by jandubois** [comment]
We talked about this today in the kubecf planning meeting, and the status is still unchanged: we don't have a `bits-service` image with a cflinuxfs3 based eirinifs in it.

There are currently no plans to work on such an image either; it would have to be contributed externally at the current time.

I hope that in the future we will be able to make stacks configurable at deployment time, without having them embedded in pre-built container images, but there is no timeline / commitment to implement that so far.

I think this issue should remain open until we have file some more actionable alternative.

**Comment by viovanov** [comment]
I think this can be closed now because of #1086
